libs/hdf-converters/test/mappers/reverse/checklist_reverse_mapper.spec.ts

```diff
@@ -88,6 +88,20 @@ describe('hdf_profile_with_invalid_metadata', () => {
   });
 });
 
+describe('hdf_profile_with_multiple_mac_host_addresses', () => {
+  it('can correctly provide multiple mac host addresses for the checklist file', () => {
+    const hdfData = loadJsonFile(
+      'sample_jsons/checklist_mapper/multiple_mac_addresses_metadata.json'
+    );
+    const mapper = new ChecklistResults(hdfData);
+    const jsonixData = mapper.getJsonix();
+    const hostmac = (jsonixData?.value as Stigdata).asset?.hostmac;
+    const expectedHostMac =
+      '02:B9:78:82:FE:DE\nEE:EE:EE:EE:EE:EE\n6E:8D:55:AB:10:5F';
+    expect(hostmac).toEqual(expectedHostMac);
```

Comment:
I fell much better if we would do something like:
const expectedHostMac ='02:B9:78:82:FE:DEEE:EE:EE:EE:EE:EE6E:8D:55:AB:10:5F';
expect(hostmac).toEqual(expectedHostMac.replaceAll('\n', ''));

Reply:
Updated :+1: